test/gateway/AxelarAmplifierGateway.js

```diff
@@ -586,6 +586,9 @@ describe('AxelarAmplifierGateway', () => {
                     .withArgs(i + 1, keccak256(encodedSigners), encodedSigners);
 
                 currentSigners = newSigners;
+
+                const timeDelay = isHardhat ? 12 * 60 * 60 : 5;
+                await waitFor(timeDelay);
```

Comment:
```suggestion
                await waitFor(5);
```
waiting more for hardhat network isn't relevant since the issue only exists on some live networks